cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -68,4 +68,8 @@ func SetDefaults_NodeConfiguration(obj *NodeConfiguration) {
 	if obj.CACertPath == "" {
 		obj.CACertPath = DefaultCACertPath
 	}
+	if len(obj.Token) != 0 {
```

Comment:
we rewrote this logic to:
```go
if len(obj.TLSBootstrapToken) == 0 {
   obj.TLSBootstrapToken = obj.Token
}
if len(obj.DiscoveryToken) == 0 {
   obj.DiscoveryToken = obj.Token
}
```

@jbeda thought that would make more sense and generally be easier and I agree